apps/nextjs/src/middlewares/auth.middleware.ts

```diff
@@ -120,6 +118,14 @@ function conditionallyProtectRoute(
       return NextResponse.redirect(new URL("/onboarding", req.url));
     }
   }
+  if (
+    userId &&
+    isOnboardingRoute(req) &&
+    !needsToCompleteOnboarding(sessionClaims)
+  ) {
+    log("Already onboarded: REDIRECT");
```

Comment:
Intentional to keep the log?

Reply:
Yep. You'll see that there are other similar logs in this file. They're under `middleware:auth` which is hidden with the default DEBUG env var